Cleanup moodle-versions file (#274)
CERT-Polska/Artemis-modules-extra: karton_moodle_scanner/moodle_versions.txt

# based on: https://github.com/inc0d3/moodlescan/blob/master/data/version.txt
# licensed under GNU General Public License v3.0
# upgraded with legacy versions

> # The list was done considering till version 4.3.12 while removing the last unique hash occurerences

Reviewer: ```suggestion
# The list is based on version 4.3.12 after removing non-unique hash occurrences.
```

> # It's done to avoid FP as often files like e.g; /privacy/export_files/general.js did not change at all during 6 years span

Reviewer: ```suggestion
# This helps avoid false positives, as some files (e.g. /privacy/export_files/general.js) have remained unchanged for over 6 years.```
